packages/mobile/src/tokens/saga.ts

```diff
@@ -248,7 +245,7 @@ export function tokenTransferFactory({ actionName, tag }: TokenTransferFactory)
         )
       } catch (error) {
         Logger.error(tag, 'Error transfering token', error)
-        yield put(removeStandbyTransactionLegacy(context.id))
+        yield put(removeStandbyTransaction(context.id))
```

Comment:
Hmm, we are creating a legacy pending tx but removing the non-legacy one?

I think for now we need to both and and remove both legacy and non-legacy so that it works weather we show the new or old feed.

Alternatively, we could use one or the other depending on the status of the feature flag for the new transaction feed

Reply:
Sorry, maybe it's a bit confusing but `removeStandbyTransaction` removes from the legacy and the non-legacy: https://github.com/valora-inc/wallet/pull/1669/files#diff-5ba66e05e1de1827234d9fbc6273e0f406fc733151024de936747ce2e77b252fR80
At first I did separately, but then I realized it was the same code duplicated and we could do it like this.
Let me know if you think isn't clear enough

Reply:
Ohh, yeah that makes a lot of sense, nevermind.